.github/workflows/publish.yml

```diff
@@ -38,7 +38,19 @@ jobs:
       # Make a release
       - run: echo TAG=${GITHUB_REF##*/} >> $GITHUB_ENV
       - run: tarantoolctl rocks new_version --tag ${{ env.TAG }}
-      - run: tarantoolctl rocks install cartridge-${{ env.TAG }}-1.rockspec
+      - run: tarantoolctl rocks make cartridge-${{ env.TAG }}-1.rockspec
+      - run: |
+          mkdir -p check-version
+          # make sure the modules are loaded from ../.rocks, but not ./
```

Comment:
Also you could verify it with "package.search('cartridge')"

Reply:
I guess the words "make sure" aren't accurate enough here. It's not necessary to check `package.search` because if it's `./cartridge.lua`, then it'll always return 'scm-1' from `cartridge/VERSION.lua` (which isn't rendered during build) and thus the assertion will always fail.

So I've changed the comment.